transaction-service/transaction-module/src/main/kotlin/com/ajaxsystems/infrastructure/mapper/TransactionMapper.kt

```diff
@@ -1,11 +1,11 @@
-package ua.com.foxminded.restClient.mapper
+package com.ajaxsystems.infrastructure.mapper
```

Comment:
com.ajaxsystems.infrastructure.database.mapper

since this mapper is used to map DB entity, it can be moved to db package